### ISSUE TITLE
Provide convenience symlinks information in `bazel info`

### ISSUE THREAD

**Issue by tpasternak** [bazelbuild/bazel#23035]
### Description of the feature request:

`bazel info` already provides a way to obtain output_base directory and some significant subdirectories.

Would it be also possible to provide similar information about the convenience symlinks? They can be manipulated with the `--symlink_prefix` flag, so we can't assume their names. 

This would be useful for tools like IntelliJ to mark these folders as excluded (from indexing) as early as possible.

### Which category does this issue belong to?

CLI

### What underlying problem are you trying to solve with this feature?

Mark the convenience symlinks as ignored as soon as possible during the IntelliJ sync process

**Comment by haxorz** [comment]
The `ConvenienceSymlinksIdentified` message in the BEP (https://bazel.build/remote/bep) has this information. Did you consider using that?

**Comment by tpasternak** [comment]
Indeed, works. However it would require yet another bazel build (empty) call. That might increase sync times by a few seconds. For IntelliJ it is important to mark folders as excluded before the actual build starts. Otherwise, there would be a race between directory scanning and exclusion marking

**Comment by haxorz** [comment]
> However it would require yet another bazel build (empty) call. That might increase sync times by a few seconds.

What did you have in mind?

```
nharmata@nharmata:~/bazel-scratch$ for i in $(seq 5); do bazel info &> /dev/null; time bazel build &> /dev/null; done

real	0m0.066s
user	0m0.019s
sys	0m0.008s

real	0m0.073s
user	0m0.014s
sys	0m0.012s

real	0m0.062s
user	0m0.011s
sys	0m0.014s

real	0m0.066s
user	0m0.021s
sys	0m0.005s

real	0m0.064s
user	0m0.012s
sys	0m0.015s
nharmata@nharmata:~/bazel-scratch$
```

Are the long no-op `bazel build` invocations you're worried about maybe due to something else? E.g. Bazel checking for changes to external repos, etc. That seems like a separate issue to me, and I'd prefer you work around that more directly. That being said, I'm pragmatic and am willing to accept a contribution of this FR as long as it's well motivated.

Fyi I'm OOO for the next 1.5 weeks so if you need an urgent response please tag someone else.

**Comment by kalvdans** [comment]
If we fix https://github.com/bazelbuild/bazel/issues/20799 all convenence symlinks would have deterministic names and thus could be added to .gitignore or similar that IDE's read. I suggest closing this as a duplicate.
